server/model/status_page.js

```diff
@@ -60,6 +60,8 @@ class StatusPage extends BeanModel {
             head.append($(escapedGoogleAnalyticsScript));
         }
 
+        head.append(statusPage.customHtml);
```

Comment:
As you stated above:

> For the XSS attack: In many cases the person with login to the status page might also edit the html files on the server itself. For the other cases a env variable could be added that disables the new setting, so the server admin can restrict the kuma admin.

Such an environment variable (=> disabling this line and the frontend) might be a good idea. Thoughts?

Reply:
I think this PR is mostly blocked on security => "Is this safe?"

Reply:
I added a check for `UPTIME_KUMA_ALLOW_CUSTOM_HTML == 1`. I kept the setting visible, so users see that this feature exist, but added a hint.